nighthawk/BUILD

```diff
@@ -1,10 +1,24 @@
 load(
     "@envoy//bazel:envoy_build_system.bzl",
     "envoy_cc_binary",
+    "envoy_cc_library",
 )
 
 package(default_visibility = ["//visibility:public"])
 
+envoy_cc_library(
+    name = "hdrhistogram_c",
+    srcs = glob(["hdrhistogram_c/src/*.c"]),
```

Comment:
I'm not a huge fan of glob based builds of non-Bazel projects, but as long as you've verified each of these globbed files is supposed to be there (and not some test or utility), then this is fine.